docs/RichTextInput.md

```diff
@@ -155,3 +113,57 @@ export const MyEditorOptions = {
 	],
 };
 ```
+
+## `toolbar`
+
+The `<RichTextInput>` component has a `toolbar` prop that accepts a `ReactNode`.
+
+You can leverage this to change the buttons [size](#api):
```

Comment:
Wrong anchor: ... `[size]`(#api): 
Could be `https://mui.com/material-ui/api/button` ?

Reply:
You're right, I removed the anchor.